packages/react-components/react-alert/src/stories/AlertIcon.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+import { Delete16Filled } from '@fluentui/react-icons';
+
+import { Alert } from '../index';
+
+export const Icon = () => (
+  <Alert
+    icon={<Delete16Filled primaryFill="red" />}
```

Comment:
```suggestion
    icon={<DeleteFilled />}
```

- The same note about scalable icons
- We should avoid usage of exact colors (`red`) in our examples as these colors are not theme aware. I see following options:
  - use `intent` to change color
  - do not have color in this example
  - use `tokens` (might be confusing in this example)

Reply:
Updated to use `DeleteRegular`